cascade/cascade.py

```diff
@@ -1342,6 +741,15 @@ def main():
         ipaddress = args.ipaddress
     logger.debug('ip address: {}'.format(ipaddress))
 
+    # set up container mode
+    if args.mode == 'docker':
+        _CONTAINER_MODE = ContainerMode.DOCKER
+    elif args.mode == 'singularity':
+        _CONTAINER_MODE = ContainerMode.SINGULARITY
+    else:
+        raise ValueError('container mode is invalid')
```

Comment:
minor: add the specified mode in the exception: `... is invalid: {}'.format(args.mode))`